servant-client/servant-client.cabal

```diff
@@ -46,9 +56,14 @@ library
     , text
     , transformers
     , transformers-compat
+    , case-insensitive
```

Comment:
If we can in fact move `parseHeaders` to GHCJS-specific stuff, this could also go into the `impl(ghcjs)` branch below (I think).